resources/views/pages/rock-the-vote-reports/show.blade.php

```diff
@@ -12,6 +12,11 @@
     <p>
         Total rows: <strong>{{$report->record_count}}</strong>
     </p>
+    @if ($report->status === 'building')
+    <p>
+        Progress: <strong>{{round(($report->current_index * 100) / $report->record_count)}}% </strong>(processed <strong>{{$report->current_index}}</strong> rows)
```

Comment:
This could be nice to pull out into a method or computed property on the model!

```php
public function getPercentageAttribute()
{
    return round(($this->current_index * 100) / $this->record_count);
}
```

```blade
Progress: <strong>{{$report->percentage}}% </strong>(processed <strong>{{$report->current_index}}</strong> rows)
```

Reply:
As is, this data is from a Rock the Vote API request to the `/GET registrant_reports/:id` endpoint, and is not displaying the info from the model. I wanted to save that change for a future PR to help keep this reviewable, but I think it'd be a sweet change to make once we do start displaying model info here.

Reply:
Oh right!